src/animators/baseAnimator.ts

```diff
@@ -37,8 +36,8 @@ export module Animators {
      */
     public static DEFAULT_EASING = "exp-out";
 
-    private _duration: number;
     private _delay: number;
+    private _duration: number;
```

Comment:
Chatted with @bluong and we are keeping underscores if there is a method that shares the same name. For example, because the getter method delay() exists, we keep _delay as a variable